nats/Cargo.toml

```diff
@@ -11,7 +11,7 @@ chrono = "0.4"
 crossbeam="0.8"
 futures = "0.3"
 log = "0.4"
-nats = "0.13"
+nats = { version = "0.16.101", package = "nats-aflowt" }
```

Comment:
What's the difference between this and just using `nats-aflowt 0.16.103`?

Reply:
it uses nats_aflowt but it's called nats inside the rust code.

Reply:
there's no need to do this anymore, I can back it out. This was before I decided to standardize on aflowt being availble as `wasmbus_rpc::anats`, which also avoids a direct dependency